components/main_menu/main_menu.jsx

```diff
@@ -341,6 +343,13 @@ class MainMenu extends React.PureComponent {
                             dialogType={MarketplaceModal}
                             text={formatMessage({id: 'navbar_dropdown.marketplace', defaultMessage: 'Plugin Marketplace'})}
                         />
+                        <Menu.ItemToggleModalRedux
+                            id='cloudPurchaseModal'
+                            modalId={ModalIdentifiers.CLOUD_PURCHASE}
+                            show={!this.props.mobile}
+                            dialogType={PurchaseModal}
+                            text={'Purchase Cloud'}
+                        />
```

Comment:
This should be removed before merge. This was just a easy access for dev/testing